### PR TITLE
Addition of "Diff" to LISA tools

### PR DESCRIPTION
**Pull request by shekharsorot**
Adding simple functionality for "diff" command in Linux based test runners. Diff allows for comparison of two files or directories.
Tool can be called via class "Diff"
Currently only "comparefiles" function is implemented,
TODO: comparedirectory is pending

### CODE REVIEW

File: lisa/tools/__init__.py [hunk 1]
```diff
@@ -1,262 +1,264 @@
...
-    "YumConfigManager",
-    "Vdsotest",
-    "VcRunner",
-    "VcTargetInfo",
-    "VirtualClientTool",
-    "Who",
-    "Whoami",
-    "Wsl",
-]
+# Copyright (c) Microsoft Corporation.
```
Comment (squirrelsc) [inline lisa/tools/__init__.py:1]: What's change the whole file? Please double check it.

File: lisa/tools/diff.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    def comparefiles(
+        self,
+        src: PurePath,
+        dest: PurePath,
+        cwd: Optional[PurePath] = None,
+        sudo: bool = False,
+        timeout: int = 600,
+    ) -> str:
+        cmd = f"{self.node.get_str_path(src)} {self.node.get_str_path(dest)}"
+        result = self.run(
```
Comment (squirrelsc) [inline lisa/tools/diff.py:61]: What's the output looks like? How the result will be used?

File: lisa/tools/diff.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    @property
+    def command(self) -> str:
+        return "diff"
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def _check_exists(self) -> bool:
+        return True
```
Comment (squirrelsc) [inline lisa/tools/diff.py:50]: Don't overwrite this method, let LISA check if it exists in default paths.

**Comment by shekharsorot** [comment]
Due to a checksum issue, _init_.py is being treated as new file. Will re-raise PR with correction

**Pull request closed without merging**